tests/scaffold/mobile-app.test.mjs

```
@@ -0,0 +1,13 @@
+import assert from 'node:assert/strict';
+import fs from 'node:fs';
+
+const root = new URL('../../', import.meta.url);
```

ESLint

[error] 'URL' is not defined.

(no-undef)